infrastructure/database/set_environment.local.sh

```diff
@@ -15,3 +15,5 @@ export DB_GDK=
 
 export QTREES_VERSION="local"
 
+export SOLARANYWHERE_API_KEY='' # needs to be single quote to escape the api keys dollar sign
```

Comment:
@birdMarcus 
- gehört das nicht auch ins private-repo?
- in 1password hinterlegt?

Reply:
In PW hinterlegt in 1password ja. 
SolarAnywhere ist ja ne öffentliche API. Daher hab ich das jetzt doch hierher gemoved. Wenn jemand nen Key hat kann jemand unseren Code ja nutzen. Wichtig sind, dass die Daten (d.h. Tabellen) im privaten Schema sind.